app/controllers/orders_controller.rb

```diff
@@ -29,6 +28,13 @@ def create
     end
   end
 
+  def destroy
+    order = Order.find(params[:id])
+    order.cancel_order
```

Comment:
Is this the whole "you don't actually destroy things, just don't show them" part of the readme?